controllers/os-coreos-alicloud/pkg/coreos-alicloud/actuator_reconcile.go

```diff
@@ -54,6 +54,13 @@ func (a *actuator) cloudConfigFromOperatingSystemConfig(ctx context.Context, con
 		files = append(files, &internal.File{Path: file.Path, Content: data, Permissions: file.Permissions})
 	}
 
+	// blacklist sctp kernel module
+	files = append(files,
```

Comment:
This file should only be written if `config.Spec.Purpose == extensionsv1alpha1. OperatingSystemConfigPurposeReconcile`.

Reply:
I don't understand. What is this extensionsv1alpha1. OperatingSystemConfigPurposeReconcile for? This needs to go in regardless.

Reply:
https://github.com/gardener/gardener/blob/master/docs/extensions/operatingsystemconfig.md

Reply:
done